### PR TITLE
Increase Loki memory limit to 3Gi

### PR DESCRIPTION
**Pull request by vlvasilev**
**How to categorize this PR?**

/area logging
/kind enhancement

**What this PR does / why we need it**:
With this PR the Loki memory limit is increased to 3Gi

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
Loki memory limit is increased to 3Gi.
```

### CODE REVIEW

**Review by rfranzke — COMMENTED** [review]
/approve

File: charts/seed-bootstrap/charts/loki/values.yaml [hunk 1]
```diff
@@ -51,7 +51,8 @@ replicas: 1
 resources:
   loki:
     limits:
-      memory: 10Gi
+#TODO (vlvasilev) after eliminating the Loki OOM Kill issue return it to 10Gi
```
Comment (ialidzhikov) [inline charts/seed-bootstrap/charts/loki/values.yaml]: By this issue do you mean a bug on our side (memory leak in the curator for example)? If yes, can you please link the issue.

```suggestion
      # TODO(vlvasilev): After fixing the following issue https://github.com/gardener/logging/issues/119, set the memory limit to 10Gi.
      memory: 3Gi
```

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

File: charts/seed-bootstrap/charts/loki/values.yaml (hunk 1 above)
Comment (vlvasilev) [reply charts/seed-bootstrap/charts/loki/values.yaml]: Still, there is no open issue. I am working on that problem now.

Comment (ialidzhikov) [reply charts/seed-bootstrap/charts/loki/values.yaml]: So is it a different issue from https://github.com/gardener/logging/issues/119? If yes, please open a new issue and link it in the TODO so we know exactly after which issue fix we are safe to increase the limit again.

**Review by ialidzhikov — APPROVED** [review]

**Pull request merged**